pkgs/by-name/an/anyrun/package.nix

```diff
@@ -73,5 +69,6 @@ rustPlatform.buildRustPackage rec {
       NotAShelf
     ];
     mainProgram = "anyrun";
+    platforms = lib.platforms.linux; # gtk-layer-shell doesn't support Darwin
```

Comment:
Shouldn't this be propagated already?

Reply:
Um, this was your request?

Reply:
It is, you currently can't build on darwin because of that dependency, it won't even evaluate due to the unsupported platform. That's why I wanted to mark darwin as a badPlatform and move gtk-layer-shell to the linux buildInputs.

Reply:
Ah, I missed the bit about moving gtk-layer-shell. Making the change.

Reply:
FYI, that change now breaks compilation on Darwin (wayland-commons fails w/ MSG_NOSIGNAL not in MsgFlags.)